google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerOptionsTest.java

```diff
@@ -382,4 +383,27 @@ public void testDoNotCacheClosedSpannerInstance() {
     assertThat(service3.isClosed()).isFalse();
     service3.close();
   }
+
+  @Test
+  public void testSetClientLibToken() {
```

Comment:
let's also add an assertion for not setting the client lib and the value returned

Reply:
That assertion is on line 402.